api/v1alpha1/sandbox_types.go

```diff
@@ -66,6 +66,9 @@ type SandboxSpec struct {
 	// PodTemplate describes the pod spec that will be used to create an agent sandbox.
 	// +kubebuilder:validation:Required
 	PodTemplate PodTemplate `json:"podTemplate" protobuf:"bytes,3,opt,name=podTemplate"`
+
+	// ShutdownAt - Absolute time (RFC 3339) when the sandbox is deleted.
+	ShutdownAt string `json:"shutdownAt,omitempty"`
```

Comment:
I suggest this field we can use struct, because current we only define shutdown action, the future may also need startup action, it like :

```
type SchedulerTime struct {
    ShutdownTime metav1.Time `json:"shutdownTime,omitempty"`
    StartupTime metav1.Time `json:"startupTime,omitempty"`
}
```